backend/src/apiserver/resource/resource_manager_util_test.go

```diff
@@ -92,7 +92,7 @@ status:
     resubmit-hl9ft-3879090716:
       boundaryID: resubmit-hl9ft
       children:
-      - resubmit-hl9ft-3895868335
+      - resubmit-hl9ft-3.105868335
```

Comment:
undo this, same with [this](https://github.com/kubeflow/pipelines/pull/12243/files#diff-629f33eea4675d8431f5c7dc7605130cb5493f68cd558c15b8a0b09607cc598dR105) and [this](https://github.com/kubeflow/pipelines/pull/12243/files#diff-629f33eea4675d8431f5c7dc7605130cb5493f68cd558c15b8a0b09607cc598dR109)